src/core/exec.rs

```diff
@@ -156,9 +157,10 @@ impl Editor {
             }
             (Command(Normal), Char('b')) => {
                 // Branch cursor
-                if self.cursors.len() < 255 {
+                if self.buffers.current_buffer_info().cursors.len() < 255 {
                     let cursor = self.cursor().clone();
-                    self.cursors.insert(self.current_cursor as usize, cursor);
+                    let current_cursor_index = self.buffers.current_buffer_info().current_cursor as usize;
+                    self.buffers.current_buffer_info_mut().cursors.insert(current_cursor_index, cursor);
                     self.next_cursor();
```

Comment:
Do you think I should move `cursor()`, `next_cursor()` and `prev_cursor()` into `BufferInfo`? If I did that, then I could pull `self.buffers.current_buffer_info_mut()` into a local. It doesn't work now because of the borrow checker.